core/vm/interpreter.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // Config are the configuration options for the Interpreter
 type Config struct {
-	Debug                   bool      // Enables debugging
+	Debug                   bool      // Opcode logger
```

Comment:
Aha if we remove this downstream code will break

```suggestion
        // Opcode logger
        // Deprecated: this field is not used internally anymore and will be removed.
	Debug                   bool
```

Reply:
:angry: meh ok